features/README.md

```diff
@@ -284,12 +333,15 @@ When creating a new step, think about parametrisation and optimisation of the st
 Before creating a new step, check if something similar already exist.
 Try to reuse existing steps.
 
-#### Do not use "when" keyword
+#### Do not use "when" or "And" keywords
 The "when" keyword must not be used.
-Allowed keywords are: `Given`, `Then`, `And`, `*`
+The "And" keyword must not be used.
```

Comment:
> The "And" keyword must not be used.

Do we currently comply with this?

Reply:
@Ghesselink has been working on categorizing the various given statements we have:

- global file-based such as `given schema ...`
- intial scope selection `Given an IfcWall`
- filtering scope `Given attribute Name = ...`
- navigating scope `Given its attribute XXX`  `Given an relation IfcX to IfcY`

I think it would be very nice to add that here, but that can of course be a subsequent PR.

Reply:
> > The "And" keyword must not be used.
> 
> Do we currently comply with this?

Yes, I refactored all the statements a while ago. Cannot find back exactly when. 
We're not enforcing it at the moment with the conventions-check.

Reply:
> I think it would be very nice to add that here, but that can of course be a subsequent PR.

Think I'll use our work at the moment as a sort of R&D for these improvements and add everything to the VS after the 11th of March.